rgd/geodata/search.py

```diff
@@ -14,7 +14,7 @@
 from rest_framework.decorators import api_view
 
 from . import serializers
-from .models import GeometryEntry, RasterEntry, SpatialEntry
+from .models import GeometryEntry, RasterMetaEntry, SpatialEntry
```

Comment:
@manthey, @mcovalt: how do you all feel about this. The changes in this PR remove the spatial reference from `RasterEntry` and into `RasterMetaEntry` so that is what has to be queried against

Reply:
This seems fine to me. Thanks for the heads up!